Creating a positive pathogen test for an event participant whose event has no disease produces an error

<!--
If you've never submitted an issue to the SORMAS repository before or this is your first time using this template, please read the Contributing guidelines (accessible in the right sidebar) for an explanation about the information we'd like you to provide.
-->
### Bug Description
Creating a positive pathogen test for an event participant whose event has no disease produces an error (NullPointer)

### System Details
* Device: PC
* SORMAS version: 1.58.0-SNAPSHOT

### Additional Information
`Caused by: java.lang.NullPointerException: null
	at de.symeda.sormas.ui.samples.PathogenTestController.showConvertEventParticipantToCaseDialog(PathogenTestController.java:231)
	at de.symeda.sormas.ui.samples.PathogenTestController.lambda$savePathogenTest$9(PathogenTestController.java:217)
	at de.symeda.sormas.ui.samples.PathogenTestController.lambda$savePathogenTest$10(PathogenTestController.java:222)
	at de.symeda.sormas.ui.samples.SampleDataView.lambda$initView$1(SampleDataView.java:159)
	at de.symeda.sormas.ui.samples.PathogenTestController.savePathogenTest(PathogenTestController.java:222)
	at de.symeda.sormas.ui.samples.PathogenTestController.lambda$getPathogenTestCreateComponent$0(PathogenTestController.java:96)
	at de.symeda.sormas.ui.utils.CommitDiscardWrapperComponent.onCommit(CommitDiscardWrapperComponent.java:525)
	at de.symeda.sormas.ui.utils.CommitDiscardWrapperComponent.commit(CommitDiscardWrapperComponent.java:392)
	at de.symeda.sormas.ui.utils.CommitDiscardWrapperComponent.commitAndHandle(CommitDiscardWrapperComponent.java:412)
	at de.symeda.sormas.ui.utils.CommitDiscardWrapperComponent$1.buttonClick(CommitDiscardWrapperComponent.java:285)
	at jdk.internal.reflect.GeneratedMethodAccessor780.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at com.vaadin.event.ListenerMethod.receiveEvent(ListenerMethod.java:712)
	... 87 common frames omitted
]]`

--- a/sormas-ui/src/main/java/de/symeda/sormas/ui/samples/PathogenTestController.java
+++ b/sormas-ui/src/main/java/de/symeda/sormas/ui/samples/PathogenTestController.java
@@ -228,7 +228,7 @@ public class PathogenTestController {
 
 	public void showConvertEventParticipantToCaseDialog(EventParticipantDto eventParticipant, Disease testedDisease) {
 		final EventDto event = FacadeProvider.getEventFacade().getEventByUuid(eventParticipant.getEvent().getUuid());
-		final boolean differentDiseases = !event.getDisease().equals(testedDisease);
+		final boolean differentDiseases = event.getDisease() == null || !event.getDisease().equals(testedDisease);
 		Label dialogContent = differentDiseases
 			? new Label(I18nProperties.getString(Strings.messageConvertEventParticipantToCaseDifferentDiseases))
 			: new Label(I18nProperties.getString(Strings.messageConvertEventParticipantToCase));

